CANTroller2/src/globals.h

```diff
@@ -426,33 +408,16 @@ float gas_pulse_ccw_max_us = 2500;  // Servo ccw limit pulsewidth. Hotrc control
 float gas_pulse_park_slack_us = 30;  // Gas pulsewidth beyond gas_pulse_idle_us where to park the servo out of the way so we can drive manually (in us)
 
 // tachometer related
-volatile int64_t tach_us = 0;
-int32_t tach_buf_us = 0;
-volatile int64_t tach_timer_start_us = 0;
-volatile int64_t tach_time_us;
-volatile int64_t tach_timer_read_us = 0;
-float tach_target_rpm, tach_adjustpoint_rpm;
-float tach_rpm = 50.0;  // Current engine speed, raw value converted to rpm (in rpm)
-float tach_filt_rpm = 50.0;  // Current engine speed, filtered (in rpm)
-float tach_govern_rpm;  // Software engine governor creates an artificially reduced maximum for the engine speed. This is given a value in calc_governor()
-float tach_convert_rpm_per_rpus = 60.0 * 1000000.0;  // 1 rot/us * 60 sec/min * 1000000 us/sec = 60000000 rot/min (rpm)
-bool tach_convert_invert = true;
-int32_t tach_convert_polarity = 1;  // Forward      
-float tach_ema_alpha = 0.015;  // alpha value for ema filtering, lower is more continuous, higher is more responsive (0-1). 
-float tach_idle_rpm = 700.0;  // Min value for engine hz, corresponding to low idle (in rpm) Note, this value is itself highly variable, dependent on engine temperature
-float tach_max_rpm = 7000.0;  // Max possible engine rotation speed
-float tach_redline_rpm = 5000.0;  // Max value for tach_rpm, pedal to the metal (in rpm). 20000 rotations/mile * 15 mi/hr * 1/60 hr/min = 5000 rpm
-float tach_margin_rpm = 15.0;  // Margin of error for checking engine rpm (in rpm)
-float tach_stop_thresh_rpm = 0.1;  // Below which the engine is considered stopped - this is redundant,
-int64_t tach_stop_timeout_us = 400000;  // Time after last magnet pulse when we can assume the engine is stopped (in us)
-int64_t tach_delta_abs_min_us = 6500;  // 6500 us corresponds to about 10000 rpm, which isn't possible. Use to reject retriggers
+Tachometer tachometer(tach_pulse_pin);
+float tach_target_rpm, tach_adjustpoint_rpm; // NOTE: adjustpoint is unused?
```

Comment:
Resolve this comment - is it unused? I see a couple lines in the run mode logic that use it. If it is being used maybe add a comment that explains what it is, and I'd consider renaming it to like cruise_adjustment_baseline_rpm or tach_initial_adjustment_rpm or something

Reply:
done